VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -81,9 +81,8 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
                 discussionThread.getIdentifier(),
                 discussionThread.getType() == DiscussionThread.ThreadType.QUESTION);
 
-        // Using application context to prevent activity leak since adapter is retained across config changes
         courseDiscussionResponsesAdapter = new CourseDiscussionResponsesAdapter(
-                getActivity().getApplicationContext(), this, discussionThread);
+                getActivity(), this, discussionThread);
```

Comment:
@bguertin any suggestions on how we should go about this ?
Cuz we do need the activity's reference instead of the app's in the adapter, for showing proper overlay msg when a task fails for whatever reason.